BetterAngelsLA/monorepo
# Expo PR Preview

Adds the capability to review PRs/Branches through Expo Go without having to deploy to TestFlight.  TestFlight is meant to be for builds that are staged for release.  Further PRs will be coming for test and release phase to app stores.

How to test:
1) Install Expo Go
2) Log into BA Account
3) Open up BA project or scan QR code
4) See if app opens up and you can log in

https://github.com/BetterAngelsLA/monorepo/assets/407393/0b52fa05-caba-4d61-88dc-4b73084471a7
